src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -18,6 +18,7 @@
           <dd i18n>{{ detailsCardData.orchestrator || 'Orchestrator is not available' }}</dd>
           <dt>Ceph version</dt>
           <dd>{{ detailsCardData.cephVersion }}</dd>
+          <cd-upgradable class="mb-2"></cd-upgradable>
```

Comment:
I think it should be part of `<dd>` dom only. Also the bottom padding got inconsistent and top padding needs to be reduced.

Reply:
I've updated the video, is it better now? @cloudbehl

Reply:
yes